app/helpers/styleProps.js

```diff
@@ -149,16 +156,16 @@ const otherProps = [
 ];
 
 export default [
-  ...spaceProps,
-  ...colorProps,
-  ...typographyProps,
-  ...layoutProps,
-  ...gridProps,
-  ...flexProps,
-  ...backgroundProps,
-  ...borderProps,
-  ...borderRadiusProps,
-  ...positionProps,
-  ...shadowProps,
-  ...otherProps,
+  ...space,
+  ...color,
+  ...typography,
+  ...layout,
+  ...grid,
+  ...flex,
+  ...background,
+  ...border,
+  ...borderRadius,
+  ...position,
+  ...shadow,
+  ...other,
```

Comment:
@prasanna1211 why are we repeating this which is already exposed to us by styled-system.
https://styled-system.com/table/
This can be used in our component to allow only those props that are given much easily?

Reply:
I will explain the reasoning in depth. Give me 5 minutes. Might be wrong but it has some significance.

Reply:
@govindsingh55  and @SamrithaS can you also look into this ? So that after lunch we can have a call and finalize this.

Reply:
I understood.